Automattic/woocommerce-payments
# feat: onboarding wizard text alignment

Fixes #

#### Changes proposed in this Pull Request









I'd like to improve the text alignment on the onboarding wizard.

#### Testing instructions





- Ensure you have the latest version of the dev tools: `cd docker/wordpress/wp-content/plugins/woocommerce-payments-dev-tools && git checkout trunk && git pull`
- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Ensure you have the "Add UPE additional payment methods" flag enabled in WCPay Dev Tools
- Go to the onboarding wizard: http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- Text should be centered better with the borders

Before:
![Screen Shot 2021-06-02 at 10 34 56 AM](https://user-images.githubusercontent.com/273592/120510504-1c737380-c38f-11eb-8a10-1721411781cd.png)

After:
![Screen Shot 2021-06-02 at 10 34 07 AM](https://user-images.githubusercontent.com/273592/120510526-209f9100-c38f-11eb-86ba-7402a363220d.png)

What has been addressed:
![Screen Shot 2021-06-02 at 10 34 56 AM](https://user-images.githubusercontent.com/273592/120510689-49c02180-c38f-11eb-8035-cb7c1b9ce386.png)


-------------------
